### PR TITLE
feat: allow user to change the font size

### PR DESCRIPTION
**Pull request by juancarlosfarah**
@spaenleh, no need to review this in detail, as we will change the graphing library. But just in case I'm doing something terribly wrong. 😅

### CODE REVIEW

**Comment by juancarlosfarah** [comment]
PS. @spaenleh, how do I deploy this branch to Netlify?

**Comment by spaenleh** [comment]
This repo deploys to GitHub Pages. It automatically deploys commits pushed on main, but you can trigger it manually for a different branch by going to "Actions" tab and selecting the "Deploy to GitHub pages" workflow. There you should be able to run the workflow for your PR branch using the drop-down on the right like described [in this article](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow#running-a-workflow). 
Hope it helps !

File: src/WordCloud/WordCloud.tsx [hunk 1]
```diff
@@ -157,40 +161,44 @@ const Wordcloud = ({
...
-              fontSize={computeFontSize(w.coef)}
-            >
-              {w.text}
-            </text>
-          ))
+      <svg visibility="hidden" style={{position: "absolute"}}>
+        {data?.map((c) => {
+            const coefs = c.words.map((word) => word.coef);
+            const minCoef = Math.min(...coefs);
+            let maxCoef = Math.max(...coefs);
```
Comment (juancarlosfarah) [inline src/WordCloud/WordCloud.tsx:168]: This should be a `const`.

File: src/App.tsx [hunk 2]
```diff
@@ -153,46 +158,95 @@ const App = () => {
...
+                const v = parseInt(event.target.value);
+                setMinFontSize(v);
+                if (v > maxFontSize) {
+                  setMinFontSize(maxFontSize);
+                } else {
+                  setMinFontSize(v);
+                }
+              }}
+            />
+          </div>
```
Comment (spaenleh) [inline src/App.tsx:194]: You could extract this to a component so you could re-use it for the min and max.

**Review by spaenleh — APPROVED** [review]
These changes look good to me, nothing looks too wrong.
I added `minFontSize` and `maxFontSize` to the dependency array of the useEffect that computes the visualisation so it re-renders otherwise it leads to collisions when the fontsizes change.

**Comment by spaenleh** [comment]
Closed as no longer relevant

**Pull request closed without merging**